conda_forge_tick/config_schema.py

```diff
@@ -40,6 +40,27 @@ class BotConfigVersionUpdatesSourcesChoice(StrEnum):
     ROS_DISTRO = "rosdistro"
 
 
+class BotConfigVersionUpdatesNVIDIA(BaseModel):
+    """
+    Dictates the behavior of the conda-forge auto-tick bot for version
+    updates using the NVIDIA source.
+    """
+
+    compute_subdir: Optional[str] = Field(
+        default=None,
+        description="For sources from `developer.download.nvidia.com/compute`, this string"
+        "defines the subdirectory in which to find the JSON blob containing metadata"
+        "about the latest releases of a package.",
+    )
+
+    json_name: Optional[str] = Field(
+        default=None,
+        description="For sources from `developer.download.nvidia.com/compute`, this string"
+        "defines the name of the package in the JSON blob containing metadata"
+        "about the latest releases of a package.",
+    )
+
+
 class BotConfigVersionUpdates(BaseModel):
```

Comment:
I assume this is the pydantic model mentioned in the checklist. Also, the documentation is out of date because this model contains parameters that are not documented here: https://conda-forge.org/docs/maintainer/conda_forge_yml/#bot